cypress/plugins/index.js

```diff
@@ -12,10 +12,33 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
+const secrets = require('../../secrets.applitools.json');
+const getBranchName = require('../../tools/utils/getBranchName');
+
+if (!process.env.APPLITOOLS_API_KEY) {
```

Comment:
Basically, this is either using the env variable when it is defined in the context of a github workflow (CI), or getting the key from a non-committed "secrets" file. See previous note/discussion question about using "secret" files in the repo, declarations in some bash shell, etc.

Reply:
If you use a `.env` file for the secret you can just use `process.env.APPLITOOLS_API_KEY` and call it a day. Might simplify things a little for you.